modules/juce_audio_plugin_client/AAX/juce_AAX_Wrapper.cpp

```diff
@@ -1393,24 +1393,28 @@ namespace AAXClasses
                 SetParameterNormalizedValue (paramID, (double) newValue);
         }
 
-        void audioProcessorChanged (AudioProcessor* processor) override
+        void audioProcessorChanged (AudioProcessor* processor, int flags) override
         {
             ++mNumPlugInChanges;
 
-            auto numParameters = juceParameters.getNumParameters();
-
-            for (int i = 0; i < numParameters; ++i)
+            if (flags & AudioProcessorListener::Flags::parametersUpdate)
             {

```

Comment:
Can't the `Name`s change in case of a preset/program change, so this also needs to check for that kind of update? (like you do in the VST3 handling below)

Reply:
I didn't understand this comment.
I didn't dive to AAX SDK again, but at least JUCE only 'updates' parameters. not program change.

Reply:
I thought I noticed in the VST3 wrapper you made changes to handle parameter names changing even on a preset change. I asked if this is also needed for AAX.

Reply:
I guess you mean the paramPreset which is a unique parameter used by JUCE for the presetName/programName?

Reply:
Yes you're right. But also 
`vstFlags |= Vst::kParamTitlesChanged;`
Which seems to be the parameters names?
Anyway I didn't dig deep so feel free to ignore if there's no concern here

Reply:
Ah I guess it's just for the preset title